content/en/docs/v2/integrity-by-identity.md

```diff
@@ -0,0 +1,43 @@
+# Notary v2 Integrity Wrapper, Attested by an Entity
```

Comment:
This will show up in the deploy preview if we add frontmatter:

```suggestion
---
title: Notary v2 Integrity Wrapper, Attested by an Entity
decsciption: ""
weight: 1000
---
```

Reply:
I mean it is there -- it's just not showing up in the menus w/o the frontmatter.

deploy preview: https://deploy-preview-26--notarydev.netlify.app/docs/v2/integrity-by-identity/